core/src/avm1/globals.rs

```diff
@@ -129,6 +129,101 @@ pub fn get_nan<'gc>(
     }
 }
 
+pub fn set_interval<'a, 'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    context: &mut UpdateContext<'a, 'gc, '_>,
+    this: Object<'gc>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    create_timer(activation, context, this, args, false)
+}
+
+pub fn set_timeout<'a, 'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    context: &mut UpdateContext<'a, 'gc, '_>,
+    this: Object<'gc>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    create_timer(activation, context, this, args, true)
+}
+
+pub fn create_timer<'a, 'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    context: &mut UpdateContext<'a, 'gc, '_>,
+    _this: Object<'gc>,
+    args: &[Value<'gc>],
+    is_timeout: bool,
+) -> Result<Value<'gc>, Error<'gc>> {
+    if activation.current_swf_version() < 6 {
+        return Ok(Value::Undefined);
+    }
```

Comment:
Despite what the docs say, `setInterval` seems to work in Flash 5 movies. I haven't tested older ones. I suspect they forgot to version-gate it in the player.